rln/Cargo.toml

```diff
@@ -35,6 +41,17 @@ tiny-keccak = "2.0.2"
 # serialization
 serde = { version = "1.0.103", default-features = false, features = ["derive"] }
 serde_json = "1.0.48"
+base64 = "0.13.0"
```

Comment:
This was added to make it easier to print logs for binary data. Will be removed before the PR is completed